buidler.config.js

```diff
@@ -6,21 +6,25 @@ module.exports = {
   defaultNetwork: 'localhost',
```

Comment:
It would be great to leave some more inline comments and documentation to this file.

For example: which are pure buidler configs and links to their documentation, and then which are our custom configs and links to our documentation